bbp/nse/morphio-python

```diff
@@ -23,4 +23,12 @@ pythonPackages.buildPythonPackage rec {
       git
       stdenv
     ];
- }
+
+    propogatedBuildInputs = [
+      pythonPackages.numpy
+    ];
+
+    preConfigure = ''
+        sed -i "s/'numpy>=1.14.1'//g" setup.py
```

Comment:
Same question as with `morph-tool`: is it OK to downgrade required `numpy` version?
Also, in this case I have not managed to make `morphio-python` recognize `numpy` during installation at all (but `propogatedBuildInputs` should at least ensure the dependency is brought by Nix when needed).